app/src/main/java/app/revanced/integrations/returnyoutubedislike/ReturnYouTubeDislike.java

```diff
@@ -394,15 +399,14 @@ private static void setSegmentedAdjustmentValues() {
                 segmentedLeftSeparatorHorizontalScaleRatio = 0.7f;
                 break;
             case "OnePlus":
-                configManufacturer = "oneplus";
+                configManufacturer = "OnePlus";
```

Comment:
Good catch, is this confirmed? I am doubting myself that I forgot to case it correctly.

Reply:
configManufacturer is only for logging, so it doesn't matter if the casing is wrong.